src/scenes/Projects/Members/List/ProjectMembersList.tsx

```diff
@@ -0,0 +1,59 @@
+import { fade, IconButton, makeStyles, Typography } from '@material-ui/core';
+import { Add } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { ProjectMemberCard } from '../../../../components/ProjectMemberCard';
+import { listOrPlaceholders } from '../../../../util';
+import { useProjectMembersQuery } from './ProjectMembers.generated';
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+  },
+  projectItem: {
+    marginBottom: spacing(2),
+  },
+  // TODO I'll put this in a separate component because I'm using this somewhere else. Will be in final PR
+  addButton: {
+    marginLeft: 'auto',
+    marginRight: spacing(1),
+    backgroundColor: palette.error.main,
+    color: palette.common.white,
+    '&:hover': {
+      backgroundColor: fade(palette.error.main, 0.5),
+    },
+  },
+}));
+
+export const ProjectMembersList: FC = () => {
+  const classes = useStyles();
+  const { projectId } = useParams();
+  const { data } = useProjectMembersQuery({
+    variables: {
+      input: projectId,
+    },
+  });
+
+  return (
+    <div className={classes.root}>
+      <Typography variant="h2" paragraph>
+        Team Members
+      </Typography>
+      <IconButton classes={{ root: classes.addButton }} aria-label="add button">
+        <Add />
+      </IconButton>
```

Comment:
- Needs to be conditional based on `project.team.canCreate`
- Since it's conditional let's just hide it while loading. And to prevent a jump after data is loaded let's just put it to the right of the title.